contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
Ahh, sorry, I missed that this was about PV.  We can reject at runtime, if we pursue this.  That said, HostPath as PV is still a terrible idea, and nobody should use it.